docs/operations/export_api.md

```diff
@@ -0,0 +1,426 @@
+    reason: pvcBound
+    status: "True"
+    type: PVCReady
+  links:
+    external:
+      cert: |-
+        -----BEGIN CERTIFICATE-----
+        ...
+        -----END CERTIFICATE-----
+      volumes:
+      - formats:
+        - format: raw
+          url: https://vmexport-proxy.test.net/api/export.kubevirt.io/v1alpha1/namespaces/example/virtualmachineexports/example-export/volumes/example-disk/disk.img
+        - format: gzip
+          url: https://vmexport-proxy.test.net/api/export.kubevirt.io/v1alpha1/namespaces/example/virtualmachineexports/example-export/volumes/example-disk/disk.img.gz
+        name: example-disk
+    internal:
+      cert: |-
+        -----BEGIN CERTIFICATE-----
+        ...
+        -----END CERTIFICATE-----
+      volumes:
+      - formats:
+        - format: raw
+          url: https://virt-export-example-export.example.svc/volumes/example-disk/disk.img
+        - format: gzip
+          url: https://virt-export-example-export.example.svc/volumes/example-disk/disk.img.gz
+        name: example-disk
+  phase: Ready
+  serviceName: virt-export-example-export
+```
+#### Archive content-type
+Archive content-type is automatically selected if we are unable to determine the PVC contains a KubeVirt disk. The archive will contain all the files that are in the PVC.
+
+```yaml
+apiVersion: export.kubevirt.io/v1alpha1
+kind: VirtualMachineExport
+metadata:
+  name: example-export
+  namespace: example
+spec:
+  source:
+    apiGroup: ""
+    kind: PersistentVolumeClaim
+    name: example-pvc
+  tokenSecretRef: example-token
+status:
+  conditions:
+  - lastProbeTime: null
+    lastTransitionTime: "2022-06-21T14:10:09Z"
+    reason: podReady
+    status: "True"
+    type: Ready
+  - lastProbeTime: null
+    lastTransitionTime: "2022-06-21T14:09:02Z"
+    reason: pvcBound
+    status: "True"
+    type: PVCReady
+  links:
+    external:
+      cert: |-
+        -----BEGIN CERTIFICATE-----
+        ...
+        -----END CERTIFICATE-----
+      volumes:
+      - formats:
+        - format: dir
+          url: https://vmexport-proxy.test.net/api/export.kubevirt.io/v1alpha1/namespaces/example/virtualmachineexports/example-export/volumes/example/dir
+        - format: tar.gz
+          url: https://vmexport-proxy.test.net/api/export.kubevirt.io/v1alpha1/namespaces/example/virtualmachineexports/example-export/volumes/example/disk.tar.gz
+        name: example-disk
+    internal:
+      cert: |-
+        -----BEGIN CERTIFICATE-----
+        ...
+        -----END CERTIFICATE-----
+      volumes:
+      - formats:
+        - format: dir
+          url: https://virt-export-example-export.example.svc/volumes/example/dir
+        - format: tar.gz
+          url: https://virt-export-example-export.example.svc/volumes/example/disk.tar.gz
+        name: example-disk
+  phase: Ready
+  serviceName: virt-export-example-export
+```
+#### Format types
+There are 4 format types that are possible:
+* Raw. The unaltered raw KubeVirt disk image.
```

Comment:
Looking at the rendered copy, the following bullets did not render properly because there has to be a newline before a list.

```suggestion

* Raw. The unaltered raw KubeVirt disk image.
```

Reply:
Weird my local renderer renders it properly. Added the newline.